src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TaskListViewModel.cs

```diff
@@ -167,8 +177,10 @@ public string FinishSectionStatus(int sectionId)
                 }
                 else if (ApplicationRouteId == SupportingApplicationRouteId)
                 {
-                    conditionsOfAcceptance2 = _qnaApiClient.GetAnswerByTag(ApplicationId, RoatpWorkflowQuestionTags.FinishCOA2Supporting).GetAwaiter().GetResult();
-                    conditionsOfAcceptance3 = _qnaApiClient.GetAnswerByTag(ApplicationId, RoatpWorkflowQuestionTags.FinishCOA3Supporting).GetAwaiter().GetResult();
+                    // TODO: TODO: APR-1193 - This is the return value for 9.3 section for Supporting Provider 
```

Comment:
There's a NotRequiredOverrides section in config where rules are configured based on previously entered answers, can this logic be moved to there?

Reply:
Yes, there was a code in the NotRequiredOverrides section in appsettings.json, which got overridden by the merge.
I have restored it.
But I intent to keep the code here as it is really reflecting the conditions of all states  

  {
      "SequenceId": "9",
      "SectionId": "3",
      "Conditions": [
        {
          "ConditionalCheckField": "Apply-ProviderRoute",
          "MustEqual": "3"
        }
      ]
    }

Reply:
See https://skillsfundingagency.atlassian.net/browse/APR-1356 
This needs to be properley resolved ASAP.
For now I will approve fior testing.